modules/API/pages/built-in-endpoints.adoc

```diff
@@ -2606,21 +2178,78 @@ Response::
 
 | `select`
 | No
-| Attributes of the selected edges to return. The parameter takes a list, which is a string of comma-separated values. If `select` is provided, the edges returned will only show the attributes provided.
+| Attributes of the selected edges to return. The parameter takes a list, which is a string of comma-separated values.
+If `select` is provided, the edges returned will only show the attributes provided.
 
 | `timeout`
 | No
 | Integer that specifies the number of seconds after which the query will time out. If the parameter is set to `0` or isn't provided, the system-wide endpoint timeout setting is applied.
 |===
 
-=== Delete an edge
+=== Retrieve edge by source, target, edge type, and discriminator
 
-[source,bash]
+[.wrap,console]
+----
+GET /graph/{graph_name}/edges/{source_vertex_type}/{source_vertex_id}/{edge_type}
+/{target_vertex_type}/{target_vertex_id}/{discriminator}
+----
+
+This endpoint allows you to retrieve an edge by its source, target, edge type, and
+xref:gsql-ref:ddl-and-loading:defining-a-graph-schema.adoc#_discriminator[discriminator].
+
+This endpoint requires the `READ_DATA` privilege on the types or attributes being queried.
+
+==== Sample request
+
+[tabs]
+====
+Single-attribute discriminator::
++
+--
+[.wrap,console]
+----
+curl -X GET 'http://localhost:9000/graph/multiedge/edges/Person/4/transfer/Person/30/2431'
+----
+--
+Composite discriminator::
++
+--
+[.wrap,console]
+----
+curl -X GET "http://localhost:9000/graph/multiedge/edges/Person/9/transfer2/Account/9/77,1117,2010-04-09%2003:57:01" <1>
```

Comment:
some expected error message will be 

wrong type:
```
"Discriminator "str2" cannot be converted into UINT.
```
```
Discriminator "str1" cannot be converted into DATETIME.
```
more/less discriminator provided
```
The number of the input discriminators is 2. But the number of discriminators in schema is 3.
```
```
The input discriminator has more fields than the specified in the schema. 3 discriminators are parsed at position 28.
```

Reply:
Usually, it is useful to document an error message if it isn't self-explanatory. For example, something like "Segmentation fault" is not explanatory and it can be helpful to provide an explanation. 

I don't think it is necessary to document error messages here, as the error messages themselves seem to be self-explanatory.